digdag-standards/src/main/java/io/digdag/standards/operator/RbOperatorFactory.java

```diff
@@ -127,16 +129,30 @@ private Config runCode(Config params)
                 mapper.writeValue(fo, ImmutableMap.of("params", params));
             }
 
-            final String ruby = params.get("ruby", String.class, "ruby");
+            List<String> ruby;
+            try {
+                // String?
+                final String path  = params.get("ruby", String.class, "ruby");
+                ruby = ImmutableList.of(path);
+            } catch (ConfigException ex) {
+                // List?
+                ruby = params.getListOrEmpty("ruby", String.class);
+                // Default
+                if (ruby.isEmpty()) {
+                    ruby = ImmutableList.of("ruby");
+                }
+            }
```

Comment:
Can you take a look the following link? That shows how to handle various types of Json types without `ConfigException` throwing/catching.
https://github.com/treasure-data/digdag/blob/master/digdag-plugin-utils/src/main/java/io/digdag/util/RetryControl.java#L59-L80

Reply:
fixed.